src/types/Post.ts

```diff
@@ -0,0 +1,11 @@
+import { Comment } from './Comment';
+import { User } from './User';
+
+export interface Post {
+  userId: number,
+  id: number,
+  title: string,
+  body: string,
+  user: User | null,
+  comments: Comment[] | [],
```

Comment:
empty array of comments is also of type `Comment[]`
```suggestion
  comments: Comment[],
```

Reply:
Done.